MapboxDirections.swift.podspec

```diff
@@ -43,6 +43,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxDirections"
 
-  s.dependency "Polyline", "~> 4.0"
+  s.dependency "Polyline", "~> 4.1"
```

Comment:
It’s customary to specify only major and minor version here. The only reason I’ve specified a patch version in the Cartfile is because Carthage aggressively builds for all platforms and the tvOS and watchOS builds were broken in v4.1.0.